BayPass/Model/Map/Agency.swift

```diff
@@ -5,21 +5,70 @@
 //  Created by Ayesha Khan on 10/20/18.
 //  Copyright © 2018 Tim Roesner. All rights reserved.
 //
-
 import UIKit
 
-struct Agency {
-    var name: String
-    var icon: UIImage
-    var routes: [Line]
+enum Agency: String {
+    case ACTransit = "AC " // AC Transit
+    case VTA = "SC" // VTA
+    case BART = "BAR" // Bart
+    case CalTrain = "CT" // Caltrain
+    case GoldenGateTransit = "GGT"
+    case Muni = "SF" // San Francisco Municipal Transportation Agency
+    case UnionCity = "UC" // Union City Transit
+    case ACE = "99" // Altamont Corridor Express
+    case SolTrans = "247" // SolsTrans
+    case SamsTrans = "sam" // SamsTrans
```

Comment:
Please change to "SamTrans" same goes for "SolTrans" these agencies do not have an additional 's'

Reply:
but "short_name": "sam", and "name": "SolTrans" is "short_name": "247", from the API

Reply:
Actually, nevermind! I thought you were talking about the string value